### PR TITLE
The Expenses page

### PR DESCRIPTION
**Pull request by Ivan-Zhdanovich**

### CODE REVIEW

File: src/app/(home)/expenses/page.tsx
Comment (AlexMakovetskiy) [inline src/app/(home)/expenses/page.tsx:1]: can you move expenses folder with files in src/app/(privateProfile) folder?

File: src/app/(privateProfile)/expenses/expenses.module.scss [hunk 1]
```diff
@@ -0,0 +1,317 @@
+.expensesPageWrap {
+	padding: 10px;
+	width: 66%;
+	height: 93%;
+	background-color: var(--color-silver-gray);
+
+	@media (max-width: 1210) {
```
Comment (AlexMakovetskiy) [inline src/app/(privateProfile)/expenses/expenses.module.scss:7]: max-width: 1210px

don't forget adding pixels

File: src/ui/button/button.module.scss [hunk 2]
```diff
@@ -188,3 +188,93 @@
...
+
+	&__disabled {
+		max-width: 244px;
+		height: 52px;
+		border-radius: 10px;
+		width: 15%;
+		color: var(--color-pure-light);
+		font-size: 2rem;
+		line-height: 2.8rem;
+		background-color: #bbbbbb;
```
Comment (AlexMakovetskiy) [inline src/ui/button/button.module.scss:213]: better use css vars

File: src/types/common/UiKitProps.ts [hunk 3]
```diff
@@ -31,3 +33,11 @@ export interface ICustomCheckbox {
 	isChecked: boolean;
 	setIsChecked: React.Dispatch<React.SetStateAction<boolean>>;
 }
+
+export type ISelectFormProps = IExpensesSelectForm;
```
Comment (AlexMakovetskiy) [inline src/types/common/UiKitProps.ts:37]: better remove this line and use IExpensesSelectForm where it nessesary

File: src/ui/input/Input.module.scss [hunk 4]
```diff
@@ -30,11 +31,13 @@
...
 
 .inputWrap__input {
+	width: 100%;
 	flex-grow: 1;
 	border: none;
 	outline: none;
 	font-weight: 400;
 	font-size: 2rem;
+	line-height: 2.8rem;
 }
```
[the remark this answers is not in the document]
Comment (Ivan-Zhdanovich) [reply src/ui/input/Input.module.scss]: done

**Review by AlexMakovetskiy — APPROVED** [review]

**Pull request merged**